src/lib/utils/restart.web.js

```diff
@@ -1,3 +1,3 @@
-export default () => {
-  window.location = '/'
+export default redirectUrl => {
+  window.location = redirectUrl || '/'
```

Comment:
@mauriguz 
1. use location.replace(url)  or location.reload(true) if no redirect url has been passed
2. we're have some other places where we're asking to reload the window (e.g. on failed FV initialization, inside AppSwitch etc). Please refactor them all to use utils/restart

Reply:
PR with these changes created